gammapy/background/models.py

```diff
@@ -650,3 +651,26 @@ def compute_rate(self):
         bg_rate = bg_rate.to('MeV-1 sr-1 s-1')
 
         self.bg_rate.data = bg_rate
+
+    def smooth(self):
+        """Smooth the bkg rate with a gaussian 1D kernel.
+
```

Comment:
Maybe add some more info to the docstring. I don't mean a full description of the algorithm, but something like:

```
Calling this method modifies the ``bg_rate`` data member, replacing it with a smoothed version.
```
and
```
This method uses an adaptive Poisson method to compute the smoothing Kernel width
from the available counts (see code and inline comments for details).
```